lib/yard/registry_resolver.rb

```diff
@@ -185,5 +185,35 @@ def collect_namespaces(object)
 
       nss
     end
+
```

Comment:
You should probably mark all these methods below as private since they are not part of the public API.

```suggestion

    private

```

Reply:
There's a `private` marker on line 93 that covers these methods, so that's already taken care of?

Reply:
You're right about that, I missed it! You can ignore this one.